openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedTab/ActivityFeedTab.component.tsx

```diff
@@ -205,7 +205,9 @@ export const ActivityFeedTab = ({
 
   const { feedFilter, threadType } = useMemo(() => {
     const currentFilter =
-      currentUser?.isAdmin && currentUser?.fullyQualifiedName === fqn
+      currentUser?.isAdmin &&
+      currentUser?.name === fqn &&
```

Comment:
```suggestion
      currentUser?.fullyQualifiedName === fqn &&
```

Reply:
It's should be `name` as there are case where username contain `special character` in it, and while validaing with fqn it will return false